terra-jupyter-base/Dockerfile

```diff
@@ -174,7 +175,6 @@ RUN chown -R $USER:users $JUPYTER_HOME \
  && cp $JUPYTER_HOME/custom/extension_entry_jupyter.js $HOME/.jupyter/custom/custom.js \
  && cp $JUPYTER_HOME/custom/safe-mode.js $HOME/.jupyter/custom/ \
  && cp $JUPYTER_HOME/custom/edit-mode.js $HOME/.jupyter/custom/ \
- && $JUPYTER_HOME/scripts/extension/jupyter_install_lab_extension.sh $JUPYTER_HOME/custom/extension_entry_jupyterlab.js \
```

Comment:
This was failing.. but I don't think we need it. We don't support JupyterLab, and even if we did we no longer need the `google_sign_in` extension (which is what this is installing I think). Also this will be changing anyway in https://github.com/DataBiosphere/terra-docker/pull/209.